File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-roster_result.ts


import { GetTrainingEventRoster_Item } from './get-training-event-roster_item';
import { IGetTrainingEventRoster_Result } from './iget-training-event-roster_result';

export class GetTrainingEventRoster_Result implements IGetTrainingEventRoster_Result {
  
	public TrainingEventID: number = 0;
	public Rosters?: GetTrainingEventRoster_Item[];
  
}

File: GTTS/INL.TrainingService.Models/GetTrainingEventRoster_Result.cs
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventRoster_Result : IGetTrainingEventRoster_Result
    {
        public long TrainingEventID { get; set; }
        public List<GetTrainingEventRoster_Item> Rosters { get; set; }
    }
}
